docs/types.rst

```diff
@@ -55,6 +55,8 @@ Operators:
 Division always truncates (it is just compiled to the ``DIV`` opcode of the EVM), but it does not truncate if both
 operators are :ref:`literals<rational_literals>` (or literal expressions).
 
+The modulus operator ``%`` yields the remainder from the division of the left operand by the right operand. Modulus results in the same sign as its left operand (or zero), for example ``int256(-5) % int256(-2) == int256(-1)`` and ``int256(5) % int256(-2) == int256(1)``, respectively.
```

Comment:
Should also give an example with a positive modulo, otherwise it gives the impression it only works with negative ones (who uses negative modulo anyway?).

Reply:
Sounds reasonable. I've switched both examples and removed the signs from the first one. This way it should be clear how ``%`` works. Rebased the branch as well.